src/main/java/org/kohsuke/github/GHTeam.java

```diff
@@ -57,7 +57,11 @@ GHTeam wrapUp(GitHub root) { // auto-wrapUp when organization is known from GET
 
     static GHTeam[] wrapUp(GHTeam[] teams, GHPullRequest owner) {
         for (GHTeam t : teams) {
-            t.root = owner.root;
+            try {
+                t.wrapUp(owner.root.getOrganization(owner.getRepository().getOwnerName()));
+            } catch (IOException e) {
+                throw new GHException("Failed to get organization that owns the repository for the PR", e);
+            }
```

Comment:
this feels odd to me. do we need the try-catch here?
also, if so, we should defn. move the `owner.root.getOrganization(owner.getRepository().getOwnerName())` outside the loop